bx_django_utils_tests/test_app/models.py

```diff
@@ -38,3 +41,21 @@ class GtinFieldTestModel(models.Model):
 class ColorFieldTestModel(models.Model):
     required_color = ColorModelField()
     optional_color = ColorModelField(blank=True, null=True)
+
+
+
+class StoreSaveModel(models.Model):
+    name = models.CharField(max_length=64)
+    _save_calls = threading.local()
```

Comment:
I assume that this is for tests, isn't it? But i didn't see any usage in the tests below?!?

Reply:
`test_model_manipulate.py` line 420. If you have a better solution to test `save` calls I'm all for it.